contrib/kind.sh

```diff
@@ -543,8 +543,8 @@ set_default_params() {
   KIND_DNS_DOMAIN=${KIND_DNS_DOMAIN:-"cluster.local"}
   KIND_CONFIG=${KIND_CONFIG:-${DIR}/kind.yaml.j2}
   KIND_REMOVE_TAINT=${KIND_REMOVE_TAINT:-true}
-  KIND_IPV4_SUPPORT=${KIND_IPV4_SUPPORT:-true}
-  KIND_IPV6_SUPPORT=${KIND_IPV6_SUPPORT:-false}
+  KIND_IPV4_SUPPORT=${PLATFORM_IPV4_SUPPORT:-true}
```

Comment:
hmm... this is a bit dangerous, @martinkennelly . In bash, one would use this trick to behave as "set this variable with a default value, unless it is already set. Please completely rename this variable to `PLATFORM_IPVX_SUPPORT` or make it smarter.

Lastly, please ensure to also make this change in the kind-helm.sh file as well. Let me know if that is not easy to perform.